src/ReadVTK.jl

```diff
@@ -443,6 +459,54 @@ function get_data(data_array::VTKDataArray{T,N,<:FormatAppended}) where {T,N}
 end
 
 
+"""
+    get\\_data\\_reshaped(data_array::VTKDataArray; cell_data=false)
+
+Retrieve actual data from a `VTKDataArray` and reshapes them as 1D, 2D or 3D arrays, in case we deal with structured grids.
+Note that vectors or tensors will have their components stored in the first dimension of the array. 
+As there is no way to automatically tell from the VTK file format whether it is a tensor, the user has to reshape this accordingly.
+
+"""
+function get_data_reshaped(data_array::VTKDataArray{T,Ncomponents}; cell_data=false) where {T,Ncomponents}
```

Comment:
```suggestion
function get_data_reshaped(data_array::VTKDataArray{T,N}; cell_data=false) where {T,N}
```
In the actual type we also use `N` for this type parameter, thus I'd prefer to stay consistent here.